include/dt-bindings/pwm/pwm.h

```diff
@@ -6,6 +6,23 @@
 #ifndef ZEPHYR_INCLUDE_DT_BINDINGS_PWM_PWM_H_
 #define ZEPHYR_INCLUDE_DT_BINDINGS_PWM_PWM_H_
 
+/**
+ * @name PWM period set helpers
+ * The period cell in the PWM specifier needs to be provided in nanoseconds.
+ * However, in some applications it is more convenient to use another scale
+ * such as milliseconds.
+ * @{
+ */
+
+/** Specify PWM period in seconds */
+#define PWM_SEC(x)	((x) * 1000000000UL)
+/** Specify PWM period in milliseconds */
+#define PWM_MSEC(x)	((x) * 1000000UL)
+/** Specify PWM period in microseconds */
+#define PWM_USEC(x)	((x) * 1000UL)
```

Comment:
Nit:
```suggestion
/** Specify PWM period in microseconds */
#define PWM_USEC(x)	((x) * 1000UL)
/** Specify PWM period in milliseconds */
#define PWM_MSEC(x)	(PWM_USEC(x) * 1000UL)
/** Specify PWM period in seconds */
#define PWM_SEC(x)	(PWM_MSEC(x) * 1000UL)
```
as I'm really bad at counting 0's

Reply:
Also, what about providing PWM_NSEC for consistency ?

Reply:
thanks for the suggestion, will add them!